lib/fika/cli.ex

```diff
@@ -9,7 +9,21 @@ defmodule Fika.Cli do
   end
 
   defp parse_args([]) do
-    :timer.sleep(:infinity)
+    if print_usage?() do
+      IO.puts("""
+      Usage:
+      <fika> start <directory>
+        Starts the router.fi file inside <directory>.
+
+      <fika> exec [filename [--function function_call]]
+        Executes the function call <function_call> inside the module defined in <filename>.
+        Defaults:
+          filename 'main.fi'
+          function: 'start()'
+      """)
```

Comment:
Shouldn't it be just `fika` instead of `<fika>`? http://docopt.org/ is a good standard for documenting CLI tools

Reply:
Nice! I'll take a look at the standard

Reply:
Changed to compatibilize with the docopt standard. Thanks for the tip!